cfg/config.yaml

```diff
@@ -12,7 +12,8 @@ llsfrb:
     game: game.log
 
   conveyor-belt:
-    enable: false
+    #mode: simulation, real, mockup
```

Comment:
For the other options we don't give the possible values. Therefore, I would not include them here. However, the CFH manual is probably a good place for them.